src/chartmuseum/templates/_helpers.tpl

```diff
@@ -54,6 +54,9 @@ Selector labels
 {{- define "chartmuseum.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "chartmuseum.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if .Values.commonLabels}}
```

Comment:
can we move these new common labels under "chartmuseum.labels"?

Reply:
As long as we have a way to deploy label on all ressources sounds good for me !

Reply:
Should be good @cbuto